package.json

```diff
@@ -9,8 +9,9 @@
   },
   "devDependencies": {
     "alt-search-docs": "1.0.6",
-    "babel": "^5.3.1",
-    "babel-core": "^5.3.1",
+    "babel": "^5.6.6",
+    "babel-core": "^5.6.6",
+    "babel-loader": "^5.1.4",
     "babelify": "^6.1.0",
     "browserify": "^10.1.3",
     "browserify-shim": "^3.8.6",
```

Comment:
I think babel-core is still needed as it is a peer dependency to babel-loader, but I am not 100%